src/main/java/com/cburch/logisim/gui/generic/AttrTable.java

```diff
@@ -429,7 +430,12 @@ public boolean shouldSelectCell(EventObject anEvent) {
     public boolean stopCellEditing() {
       // Tells the editor to stop editing and accept any partially
       // edited value as the value of the editor.
+      if (stoppedCellEditing) {
+        return false;
+      }
+      stoppedCellEditing = true;
       fireEditingStopped();
+      stoppedCellEditing = false;
```

Comment:
I recommend that you explain how the `stoppedCelleEditing` flag helps prevent the double warning issue.